sympy/solvers/tests/test_solveset.py

```diff
@@ -808,6 +811,29 @@ def test_solveset():
                                                   S.Integers)
 
 
+def test_conditonset():
```

Comment:
Not sure what happened between this PR passing the tests and this being merged, but this test is now failing in master.